test.disabled/ejabberd_tests/tests/vcard_SUITE.erl

```diff
@@ -144,6 +151,15 @@ init_per_group(ldap_only, Config) ->
         _ ->
             {skip, "this group is only for ldap vCard backend"}
     end;
+init_per_group(ro_rsm, Config) ->
+    VCardConfig = ?config(mod_vcard, Config),
+    case proplists:get_value(backend, VCardConfig) of
+        Backend when Backend =:= undefined;
+                     Backend =:= mnesia ->
+            Config;
+        _ ->
+            {skip, "RSM support is only implemented for the mnesia backend"}
```

Comment:
That's fine for now. Also, I think MongooseIM should return an error to any Vcard queries with RSM when the backend is different than `mnesia`. Otherwise, clients may get confusing results.